src/cf.data.pre

```diff
@@ -6454,9 +6454,16 @@ DOC_START
 	to be erroneously cached unless the application designer
 	has taken the appropriate actions.
 
-	'Percent' is a percentage of the objects age (time since last
-	modification age) an object without explicit expiry time
-	will be considered fresh.
+	'Percent' is used to compute the expiration time of responses with a
+	Last-Modified header and without an explicit expiry time: When Squid
+	caches (or updates) such a response, the expiration time is effectively
+	set to the given percentage of the time passed since last modification.
+	For example, if such an object was cached 1000 minutes after its
+	Last-Modified time and percent is set to 20%, then the object will be
+	considered fresh for the first 200 minutes (1000*0.20=200) it spends in
+	the cache. The lm-factor term used in the algorithm sketch below is object
+	age divided by stable-time, where stable-time is the time between the last
+	object modification and the moment the object was cached (or updated).
 
```

Comment:
I am not sure the whole algorithm sketch below and its `lm-factor` line in particular can be correctly interpreted by most admins. To avoid broadening this PR scope to the whole refresh_pattern documentation rewrite (which I cannot volunteer for right now), I decided to leave that algorithm sketch untouched in this PR and just document what its `lm-factor` term is. 

Ideally, the algorithm sketch should be moved much higher (and described a bit better) because many admins do not realize that it determines critical heuristics application order (and some probably stop reading after hitting the long list of override options/flags!). Again, I decided not to touch that for now.

I will quote that algorithm sketch here for convenience:

```
	Basically a cached object is:

		FRESH if expire > now, else STALE
		STALE if age > max
		FRESH if lm-factor < percent, else STALE
		FRESH if age < min
		else STALE
```